node/src/components/consensus/era_supervisor.rs

```diff
@@ -188,19 +190,24 @@ where
     fn booking_block_height(&self, era_id: EraId) -> u64 {
         // The booking block for era N is the last block of era N - AUCTION_DELAY - 1
```

Comment:
Is the `- 1` missing in the code? Or is `era_id` the one _before_ the era we're asking for?

Reply:
`era_id` is the era we're asking for. We're calculating the era `N - AUCTION_DELAY`, taking the starting block of that, and then subtracting 1 from _that_, which gives us the final block of era `N - AUCTION_DELAY - 1`.

Reply:
Ah, right, thanks!